core/src/main/web/app/helpers/core.js

```diff
@@ -1033,6 +1053,22 @@
             callback(result);
           }
         });
+      },
+      colorize: function (theme) {
+        var colorizedElements = $("html");
+        var ca = colorizedElements.attr('class');
+        var classes = [];
```

Comment:
I think we don't need this line anymore since we can just initialize it when we do the split